### DISCUSSION TITLE
非微调，从头开始训练合成的结果是噪音

### DISCUSSION THREAD

**Discussion by GwsinTracy** [fishaudio/Bert-VITS2#367] [General]
同样的数据集，微调合成的结果正常；同样的数据集（20h），没有用底模（代码跟微调的一样，没有改动，只是没有放预训练模型在目录中），训练集loss也在正常的下降，但是合成的结果全是噪音，，这可能是什么原因导致的？

**Comment by zazachao** [comment]
我也是这样的，现在在寻找原因，朋友，你知道怎么回事吗

**Comment by LilHOliang** [comment]
两位朋友，可以请教一下怎么从头训练吗，我也想试试从头开始训练 @GwsinTracy @zazachao

**Comment by tujie-jiangye** [comment]
同问
